src/achievementsApp/reducer.js

```diff
@@ -25,5 +26,6 @@ export default combineReducers({
   path,
   paths,
   problem,
-  fetchDataDemo
+  fetchDataDemo,
+  fusionChartDemo
```

Comment:
@bensjx the reducers and action creators are pretty the same as the ones in fetchDataDemo? in that case there is no need to duplicate another set of reducer-actions in your Ben folder